services/src/main/java/org/jboss/windup/web/services/model/MigrationProject.java

```diff
@@ -33,6 +35,7 @@
  * @author <a href="http://ondra.zizka.cz/">Ondrej Zizka, zizka@seznam.cz</a>
  */
 @Entity
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = "title", name = "uniqueTitle"))
```

Comment:
That is a good point, but I'm kind of ok with having a mediocre error for this at the moment just to keep from having duplicate titles in. It will still show up as a general error message on form save, right?